server/models/followers.js

```diff
@@ -0,0 +1,43 @@
+const getFollowersModel = (sequelize, DataTypes) => {
+  const flags = {
+    freezeTableName: true
+  };
+
+  const FellowersSchema = {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      allowNull: false,
+      defaultValue: DataTypes.UUIDV4,
+      unique: true
+    },
+
+    authorId: {
+      type: DataTypes.UUID,
+      allowNull: false
+    },
```

Comment:
I think that since `authorId` is only associated with an article and it references `userId`, it would be better if this is renamed `userId`. This is because a user does not follow an article but follows another user. Also, remember that every user is by default an author.